### PR TITLE
Fix bug where "Installed Operators" nav item isn't correctly activated

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes https://jira.coreos.com/browse/CONSOLE-1876

Screen Shot 2019-11-13 at 12 49 34 PM
Screen Shot 2019-11-13 at 12 49 58 PM
Screen Shot 2019-11-13 at 2 53 53 PM

### CODE REVIEW

**Comment by rhamilto** [comment]
/hold

this is bug

**Comment by spadgett** [comment]
Hm, I think we might have broken existing links with the 4.3 changes.

https://github.com/openshift/console/blob/release-4.2/frontend/public/components/nav/admin-nav.tsx#L84-L88

**Comment by spadgett** [comment]
The link here uses the resource plural instead of the full reference:

https://github.com/spadgett/console/blob/734c751b82d3a2f5142a57b7845bf83eb5a9b5db/frontend/packages/operator-lifecycle-manager/src/components/index.tsx#L120-L126

**Comment by spadgett** [comment]
OK, I see that we support either the plural or full reference

https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/plugin.tsx#L108-L125

File: frontend/packages/operator-lifecycle-manager/src/plugin.tsx [hunk 1]
```diff
@@ -80,6 +80,7 @@ const plugin: Plugin<ConsumedExtensions> = [
       componentProps: {
         name: 'Installed Operators',
         resource: referenceForModel(models.ClusterServiceVersionModel),
+        startsWith: 'clusterserviceversions',
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/plugin.tsx]:
```suggestion
        startsWith: ['clusterserviceversions'],
```

**Comment by spadgett** [comment]
/kind bug

Feel free to cancel the hold if ready

**Pull request merged**